### ISSUE TITLE
Module not found: Can't resolve 'node_modules/@headlessui/react/dist/components/tab/tab

### ISSUE THREAD

**Issue by acayseth** [tailwindlabs/headlessui#2679]
What package within Headless UI are you using?
Tab, Transition - @headlessui/react

What version of that package are you using?
"@headlessui/react": "^1.7.17"

What browser are you using?
Chrome, Firefox, or N/A

To reproduce
Use headlessui for transition with latest Tailwind, Nextjs

Describe your issue
```typescript
import {Tab} from "@headlessui/react";
export default function ComponentName() {
  return (
    <Tab.Group>
         <Tab.List>
           <Tab>tab_1</Tab>
           <Tab>tab_2</Tab>
           <Tab>tab_3</Tab>
         </Tab.List>
         <Tab.Panels>
           <Tab.Panel>tab_1_panel<Tab.Panel>
           <Tab.Panel>tab_2_panel<Tab.Panel>
           <Tab.Panel>tab_3_panel<Tab.Panel>
         </Tab.Panels>
    </Tab.Group>
  )
}
```
- error ./node_modules/@headlessui/react/dist/headlessui.esm.js:2:0
Module not found: Can't resolve '/home/acayseth/Develop/photolamus/new-project-template/node_modules/@headlessui/react/dist/components/tab/tab'

Import trace for requested module:
Attempted import error: 'Transition' is not exported from 'modularize-import-loader?name=Transition&join=./components/transition/transition!@headlessui/react' (imported as 'Transition').

Attempted import error: 'Tab' is not exported from 'modularize-import-loader?name=Tab&join=./components/tab/tab!@headlessui/react' (imported as 'Tab').

**Comment by mauricedesaxe** [comment]
I have a similar situation myself:

```
./node_modules/@headlessui/react/dist/headlessui.esm.js
Module not found: Can't resolve '/Users/lazaralex/Developer/data-display/node_modules/@headlessui/react/dist/components/transition/transition'

https://nextjs.org/docs/messages/module-not-found
```

and am using the same version: `"@headlessui/react": "^1.7.17",`.

Interestingly, my IDE doesn't flag the import { Dialog, Menu, Transition } from '@headlessui/react';` as bad. And when I click on `Transition`, well, it's there...

I tried forcing it to version `1.7.15` and the same issue still happens.

**Comment by thecrypticace** [comment]
Duplicate of #2677 — we'll track this there.

TL;DR it's a Next.js problem — rollback Next.js to 13.4.17 until they've released a fix
